railties/lib/rails.rb

```diff
@@ -48,13 +49,11 @@ def configuration
     end
 
     def backtrace_cleaner
-      @backtrace_cleaner ||= begin
-        # Relies on Active Support, so we have to lazy load to postpone definition until Active Support has been loaded
-        require "rails/backtrace_cleaner"
```

Comment:
I moved this `require` to the top of the file so that the `Rails::BacktraceCleaner` constant will be loaded for test_helper.rb.  The lazy load here is [over 10 years old](https://github.com/rails/rails/commit/9cfd1d44915f4615bbb760198cd01bf4dfc69f5a#diff-4d6979942db8e18c6397d28d1574d77aR703-R704), and I don't think it's relevant anymore.